lib/kramdown-rfc2629.rb

```diff
@@ -494,13 +500,18 @@ def convert_img(el, indent, opts) # misuse the tag!
         if a = el.attr
           alt = a.delete('alt').strip
           alt = '' if alt == '!' # work around re-wrap uglyness
-          if anchor = a.delete('src')
-            a['target'] = anchor
+          if src = a.delete('src')
+            a['target'] = src
           end
         end
         if alt == ":include:"   # Really bad misuse of tag...
+          anchor = el.attr.delete('anchor')
+          if !anchor
+            warn "*** missing anchor for '#{src}'"
+            anchor = src
+          end
           to_insert = ""
-          anchor.scan(/(W3C|3GPP|[A-Z-]+)[.]?([A-Za-z_0-9.-]+)/) do |t, n|
+          src.scan(/(W3C|3GPP|[A-Z-]+)[.]?([A-Za-z_0-9.\/\+-]+)/) do |t, n|
```

Comment:
```
 play.xml: Line 102: Syntax of value for attribute target of xref is not valid
 play.xml: Line 796: Syntax of value for attribute anchor of reference is not valid
```

102: `interest both in the Web graphics community [TypedArray] = <xref target="Typed+Array"/> and in`

796: `<reference anchor="Typed+Array" target="https://www.khronos.org/registry/typedarray/specs/1.0/">`

So, a + sign cannot be part of an XML anchor, but of course it can be part of a reference URL.

More importantly, putting a '/' in the reference URL confuses the caching logic as only the base name is used for the filename.